sdk/include/reactos/rosctrls.h

```diff
@@ -581,3 +581,254 @@ class CTreeView :
     }
 
 };
+
+class CTooltips :
+    public CWindow
+{
+public: // Configuration methods
+
+    HWND Create(HWND hWndParent, DWORD dwStyles = 0, DWORD dwExStyles = 0)
```

Comment:
Why do you need to override create?

Reply:
I copied from another one that did the same.